package.json

```diff
@@ -13,11 +13,12 @@
     "install-locks": "lerna exec --no-sort -- npm i --package-lock-only --no-audit",
     "lint": "eslint . && lerna run python-lint",
     "lint-md": "markdownlint docs/**/*.md docs/*.md tf-modules/**/*.md tf-modules/*.md",
-    "test": "nyc lerna run --concurrency 1 test",
-    "bootstrap": "lerna bootstrap --no-ci --force-local",
-    "bootstrap-no-build": "lerna bootstrap --no-ci --force-local --ignore-scripts",
-    "bootstrap-no-build-no-concurrency": "lerna bootstrap --no-ci --force-local --ignore-scripts --concurrency 1",
-    "bootstrap-no-build-quiet": "lerna bootstrap --loglevel=error --no-ci --force-local --ignore-scripts",
+    "test": "npm run tsc && nyc lerna run --concurrency 1 test",
+    "tsc": "lerna run tsc",
+    "bootstrap": "lerna bootstrap --no-ci --force-local --ignore-scripts && lerna run tsc && lerna run prepublish && lerna run prepare",
```

Comment:
Why is bootstrap now doing `prepublish` and `prepare` as well? I thought `lerna bootstrap` ran those anyway